src/main/scala/is/hail/HailContext.scala

```diff
@@ -511,15 +511,17 @@ class HailContext private(val sc: SparkContext,
     forceBGZ: Boolean = false,
     headerFile: Option[String] = None,
     nPartitions: Option[Int] = None,
-    sitesOnly: Boolean = false): GenericDataset = {
-    importVCFsGeneric(List(file), force, forceBGZ, headerFile, nPartitions, sitesOnly)
+    sitesOnly: Boolean = false,
+    callFields: Option[Array[String]] = None): GenericDataset = {
```

Comment:
Minor: would prefer to just make this a set?  And then check `s == "GT" || callFields.contains(s)`?  Seems a little cleaner.